bedevere/util.py

```diff
@@ -76,6 +86,15 @@ async def issue_for_PR(gh, pull_request):
     """Get the issue data for a pull request."""
     return await gh.getitem(pull_request["issue_url"])
 
+async def patch_body(gh, pull_request, issue_number):
+    """Updates the description of a PR with the gh issue number if it exists.
+
+    returns if body exists with issue_number
+    """
+    if "body" in pull_request and pull_request["body"] is not None and issue_number in pull_request["body"]:
```

Comment:
A simple "in" check could have false positives. Should probably check for `f"GH-{issue_number}\n"` or similar.

Reply:
Fixed.